addon/components/o-s-s/icon.ts

```diff
@@ -0,0 +1,31 @@
+import Component from '@glimmer/component';
+import { assert } from '@ember/debug';
+
+export type IconType = 'solid' | 'regular' | 'light' | 'duotone' | 'brand';
+
+interface OSSIconArgs {
+  icon: string;
+  type?: IconType;
```

Comment:
nitpick: `style` instead of `type` as FontAwesome calls it ? :p

Reply:
my bad 😂